t/discovery/consul.t

```diff
@@ -781,3 +781,179 @@ location /sleep {
     qr//
 ]
 --- ignore_error_log
+
```

Comment:
This file is too long, you should create a new test file

Reply:
done